aspnet/mvc/views/tag-helpers/intro.rst

```diff
@@ -112,9 +112,15 @@ You must apply the Tag Helper opt-out character to the opening and closing tag.
 Using ``@tagHelperPrefix`` to make Tag Helper usage explicit
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The ``@tagHelperPrefix`` directive allows you to specify a tag prefix string to enable Tag Helper support and to make Tag Helper usage explicit. In the code image below, the Tag Helper prefix is set to "th:", so only those elements using the prefix "th:" support Tag Helpers (Tag Helper-enabled elements have a distinctive font). The ``<label>`` and ``<span>`` elements have the Tag Helper prefix and are Tag Helper-enabled, while the ``<input>`` element does not.
+The ``@tagHelperPrefix`` directive allows you to specify a tag prefix string to enable Tag Helper support and to make Tag Helper usage explicit. In the code below, the Tag Helper prefix is set to ``th:``, so only those elements using the prefix ``th:`` support Tag Helpers (Tag Helper-enabled elements have a distinctive font). The ``<label>`` and ``<span>`` elements have the Tag Helper prefix and are Tag Helper-enabled, while the ``<input>`` element does not.
 
-.. image:: intro/_static/thp.png 
```

Comment:
Should update image instead of inlining code. A user can't visibly see what's being marked as `TagHelper` and what's not without the TagHelper highlighting.